CMakeLists.txt

```diff
@@ -77,25 +77,31 @@ set_optimization()
 ## External dependencies
 ## Dynawo and sub-dependencies
 check_env_var(DYNAWO_HOME) # Correspond to deploy installation
+get_filename_component(DYNAWO_HOME ${DYNAWO_HOME} ABSOLUTE)
 list(APPEND CMAKE_PREFIX_PATH "${DYNAWO_HOME}")
 list(APPEND CMAKE_PREFIX_PATH "${DYNAWO_HOME}/cmake")
 list(APPEND CMAKE_PREFIX_PATH "${DYNAWO_HOME}/share")
 list(APPEND CMAKE_PREFIX_PATH "${DYNAWO_HOME}/share/cmake")
 list(APPEND CMAKE_MODULE_PATH "${DYNAWO_HOME}/cmake")
 list(APPEND CMAKE_MODULE_PATH "${DYNAWO_HOME}/share/cmake")
 
-#LibArchive
-find_package(LibArchive 2.8.0 REQUIRED)
-#ZLIB
+# ZLIB
 find_package(ZLIB 1.2.3 REQUIRED)
+set_property(TARGET ZLIB::ZLIB APPEND PROPERTY
```

Comment:
I don't get the point of this line. Does find_package already say that ZLIB is found in ZLIB_LIBRARY

Reply:
The target in not well formed and we need imported location to be valid to link with a static distribution of dynawo (same is done in dynawo)